templates/docs/examples/patterns/image/container/highlighted.html

```diff
@@ -1,10 +1,10 @@
 {% extends "_layouts/examples.html" %}
+{% from "_macros/image-container.jinja" import image_container %}
+
 {% block title %}Image Container / Highlighted{% endblock %}
 
 {% block standalone_css %}patterns_image{% endblock %}
 
 {% block content %}
-<div class="p-image-container is-highlighted">
-  <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/9b4c074f-Kernelt%20industries-80-short.png" alt="" width="250">
-</div>
+  {{ image_container('https://assets.ubuntu.com/v1/9b4c074f-Kernelt%20industries-80-short.png', width='250', alt='Sample image', is_highlighted=true) }}
```

Comment:
Please don't create macros for components yet. We don't need them at this point, and can be problematic in some cases where devs are used to do things with more freedom.

For the image case, devs are almost always using image module (https://github.com/canonical/canonicalwebteam.image-template) to create the image element. We can't create a macro that doesn't support that.

So if we want to provide one it would have to take this into account, and it's way outside the scope of this PR.

Reply:
Altered to pass in image content slot rather than image properties, and no longer using image container macro